cl/opinion_page/templates/view_opinion.html

```diff
@@ -394,6 +394,8 @@ <h4>Summary of Opinion</h4>
                                     {{ sub_opinion.html_columbia|safe }}
                                 {% elif sub_opinion.html_lawbox %}
                                     {{ sub_opinion.html_lawbox|safe }}
+                                {% elif sub_opinion.html_harvard %}
```

Comment:
I assume you've taken a look at this and it looks good, great, or at least passable?

Reply:
Yes, It views normally to my eyes and my computer.